buildimage.go

```diff
@@ -53,6 +53,7 @@ func extraBinMust(n string) string {
 	return p
 }
 func main() {
+	goenv := os.Getenv("GOPATH")
```

Comment:
this will work but it makes us more dependent on GOPATH. The whole point is to build outside GOPATH.